### PR TITLE
Snappy

### PR DESCRIPTION
**Pull request by t-mon**

### CODE REVIEW

File: guh.pro [hunk 1]
```diff
@@ -33,3 +33,28 @@ contains(DEFINES, GPIO433){
 } else {
     message("Radio 433 for GPIO's disabled")
 }
+
+ubuntu {
```
Comment (mzanetti) [inline guh.pro]: maybe we should rename this to "snappy", given that all the rest is tailored to run on ubuntu/debian too.

File: guh.pro [hunk 2]
```diff
@@ -33,3 +33,28 @@ contains(DEFINES, GPIO433){
...
+    message("Building Ubuntu snappy package.")
+    meta.files = meta/package.yaml \
+                 meta/readme.md \
+                 meta/guh-logo.svg \
+                 meta/license.txt
```
Comment (mzanetti) [inline guh.pro]: I think those files should not be part of this repostiroy. They hold packaging information for the complete guh software stack, not just guhd. Maybe directly put them into a new repository, along with the packaging script.

File: guh.pro [hunk 3]
```diff
@@ -33,3 +33,28 @@ contains(DEFINES, GPIO433){
...
+    wrapper.files = meta/guhd-wrapper.sh
+    wrapper.path = /usr/bin/
+
+    # We need to bring our own Qt libs, at least for now
+    qtlibs.files=/usr/lib/arm-linux-gnueabihf/libQt5Network.so.5 \
+                 /usr/lib/arm-linux-gnueabihf/libQt5Sql.so.5 \
+                 /usr/lib/arm-linux-gnueabihf/libQt5Core.so.5 \
+                 /usr/lib/arm-linux-gnueabihf/libicui18n.so.52 \
+                 /usr/lib/arm-linux-gnueabihf/libicuuc.so.52 \
+                 /usr/lib/arm-linux-gnueabihf/libicudata.so.52
```
Comment (mzanetti) [inline guh.pro]: I don't think this is the final solution here... but don't have a better idea atm... maybe try thinking of a way to find that dynamically (ldd or something) and extracting them from the cross build chroot. For now we can leave it like this I'd say...

File: meta/package.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: guh
+version: 0.1.0
+vendor: Simon Stürz <simon.stuerz@guh.guru>
+architecture: [armhf]
```
Comment (mzanetti) [inline meta/package.yaml]: we should try to create multiarch packages... but that can be a different branch too.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/288/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/289/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/297/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/298/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/299/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/301/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/302/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/346/
Test PASSed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/347/
Test FAILed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/348/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/365/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/366/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/369/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/376/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/382/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/383/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/408/
Test PASSed.

**Comment by t-mon** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/497/

**Comment by t-mon** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/498/

**Comment by t-mon** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/499/

**Pull request merged**